Pages/Incoming.cshtml.cs

```diff
@@ -23,11 +23,10 @@ public IncomingModel(swaggerClient client, GitHubClient github)
         public IReadOnlyList<IncomingRepo> IncomingRepositories { get; private set; }
         public RateLimit CurrentRateLimit { get; private set; }
 
-        public async Task OnGet(int channelId, string repo)
+        public async Task OnGet(int channelId, string owner, string repo)
         {
-            // The 'repo' is URL encoded (because it's a URL within a URL). Decode it before calling the API.
-            var decodedRepo = WebUtility.UrlDecode(repo);
-            var latest = await _client.GetLatestAsync(decodedRepo, null, null, channelId, null, null, false, ApiVersion10._20190116);
+            var repoUrl = $"https://github.com/{owner}/{repo}";

```

Comment:
You could do something like this too. 

What I was suggesting is that you construct your URL like this:
```
https://dependencyflow.azurewebsites.net/incoming/3?repo=https%3A%2F%2Fgithub.com%2Faspnet%2FAspNetCore
```